### ISSUE TITLE
about the loss function

### ISSUE THREAD

**Issue by VeroHU** [mhaut/images-superresolution#1]
dear mhaut, 
    I have read your paper. It is really interesting to get HR images from generative model. Here I have the question:  I notice that you use the down-sampled version of the generated HR images to calculate the loss function.  That means, the function minimizes the loss between original LR and the LR version of  generated HR. However, HR is an ill-posed problem, which has many many different HR for one LR. Therefore, how can we get the correct HR (most similar to the groundtruth) when we minimize the LR version of them? I have reproduced your idea with a three-layer cnn (x4 upscaling, 3000 iterations) and the result is not very good (the left one). So I am wondering what might be the reason?
![parkinglot19_scaled](https://user-images.githubusercontent.com/30492735/48899615-08c82780-ee8b-11e8-89e2-28074deef755.jpg)![air_jpg](https://user-images.githubusercontent.com/30492735/48899806-8a1fba00-ee8b-11e8-992b-a6a261e342de.jpg)

**Comment by mhaut** [comment]
Dear VeroHU,
As any neural network, we must minimize an error or loss function at the end of the net in order to optimize the parameters of the model. However, as we do not have original HR-images, the proposed must do a down-sampling of the obtained HR in order to calculate the loss taking into account the available information, i.e. the original LR-images. Other loss functions can be used, instead the MSE, that even could produce better results. However, as a first approximation, the MSE is enough.

**Comment by VeroHU** [comment]
Hi Mhaut, I have tested the model using average-pooling as down-sampler and I got good results. Therefore, I think maybe bicubic down-sampler is not suitable in this case (as I have tested before, it tends to generate blurry images, but I don't know why). Anyway, This unsupervised idea in your paper is really innovative! Thank you!
